src/sardana/tango/core/util.py

```diff
@@ -914,14 +914,14 @@ def prepare_logstash(args):
     log_messages = []
 
     try:
-        import logstash
+        from logstash_async.handler import AsynchronousLogstashHandler
     except ImportError:
-        msg = ("Unable to import logstash. Skipping logstash "
+        msg = ("Unable to import logstash_async. Skipping logstash "
                + "configuration...", )
         log_messages.append(msg,)
         return log_messages
 
-    def get_logstash_conf(dev_name):
+    def get_logstash_conf(dev_name, class_name=None):
```

Comment:
I think this `class_name` is not used anymore?

Reply:
You're right. It was used to determine correct default DB path, but there is no default DB now.